beaconChain/tests/eth1/deploy.ts

```diff
@@ -0,0 +1,72 @@
+import BN from "bn.js";
+import { assert } from "chai";
+import { ethers } from "ethers";
+import ganache from "ganache-core";
+import sinon from "sinon";
+
+import { Eth1Notifier } from "../../src/eth1";
+import defaults from "../../src/eth1/defaults";
+import promisify from "promisify-es6";
+
+// TODO integrate this into longer running tests
+describe("", () => {
+  const N = 10; // number of private keys to generate
+  const privateKeys = Array.from({length: N}, (_, i) =>
```

Comment:
we could use ethers, but this is pretty quick and dirty. Perhaps add reference to where you got this?

Reply:
Yea I was thinking this would be easier. Originally, I was testing this with N=65537ish so I wanted the whole process to be faster than eg: using ethers to generate that many random private keys.
These private keys get plugged into an ephemeral `ganache` chain, so we can load each account up with ether (to deposit), and I figured it might even be useful to have "known" private keys around for testing.
This is all original code. :sunglasses: